ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -1492,6 +1489,10 @@ def test_frontend_method(
         shallow=False,
     )
 
+    create_frontend_array = importlib.import_module(
```

Comment:
Why is creation frontend function needed here?

Reply:
Because it needs to be passed to `flatten_frontend_to_np` below.